src/components/AccessibilityGuide/StickyElements.mdx

```diff
@@ -0,0 +1,109 @@
+
+## General Information
+
+Sticky elements switch between relative and fixed position. They move with the viewport as the user scrolls, so they are always visible (until they hit a given breakpoint).
+
+Generally, we want to avoid using sticky elements. They should be used only for critical information that needs to remain visible, or to allow a continuous process for the user.
+The most common use case, a sticky header, is only useful when users will be repeatedly accessing the menu and scrolling quickly through multiple pages. This is why you often see
+them used on e-commerce sites. If users don't need immediate navigation and will generally stay on one or two pages, a sticky header is unnecessary. Additionally, sticky headers can
+obscure other elements on the page. If you need a large menu that will inevitably cover parts of the page, consider an accordion or just making it fixed.
+
+## General Issues
+
+If you absolutely have to use a sticky element on a page, consider the following UX issues:
+
+- How much space is it taking up?
+- Is it difficult to access when tabbing forward and backwards?
+- Does it break when users zoom in on mobile devices?
+- Is it hiding focused elements?
+
+The solution presented below only addresses the _last_ of these concerns: preventing a sticky
+header from hiding focused elements. For the Digital Collections site, we chose to accept the shortcomings of the sticky header
+to maintain parity with the experience and feel of the old site.
+
+## Implementation
+
+Sighted keyboard users tab through a page to display more content and to access buttons and links.
+To return to the top of the page, keyboard users tab backwards. If the header on a page uses `position: sticky`,
+tabbing backwards moves the focused element underneath the header, so the user can no longer see where the focus is.
+A similar issue arises with users tabbing forwards against a sticky footer.
+
+The apparent solution would be using `scroll-margin-top` to snap-scroll the viewport to wherever the focused
+element is, but that's not supported on several major browsers.
+
+Instead, our approach uses a combination of CSS, HTML, and Typescript to check whether a focused element needs a sticky margin by
+dynamically calculating boundary interactions, then applies that margin (pushing page up) accordingly. This React hook is called by a
+focus event on any page element or a ResizeObserver event on the sticky element.
+
+This approach was first implemented with the Digital Collections header, using a [custom hook](https://github.com/NYPL/digital-collections/blob/main/src/hooks/useScrolled.ts) on the header.
+
+This code can be modified to apply to any sticky element, alongside
+
+1. HTML data attributes on the sticky element, ex. `<header data-sticky-header> ... </header>`
+2. CSS: `position: sticky` on the sticky element
+
+The hook first defines the `sticky` object:
+
+<Source
+  code={`
+let sticky: {
+header?: HTMLElement | null;
```

Comment:
Can you indent the code snippet here and below? It doesn't render well in storybook and makes it hard to read.